client/cd-it8.c

```diff
@@ -285,6 +285,7 @@ cd_util_create_cmf (CdUtilPrivate *priv,
 	if (dot != NULL)
 		*dot = '\0';
 	cd_it8_set_title (cmf, title);
+	cd_it8_set_enable_created (cmf, FALSE);
```

Comment:
I think this is sometimes very useful. Could we perhaps control this with a command line switch?